app/src/main/java/com/example/kursaa/features/characters/presentation/CharacterViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.kursaa.features.characters.presentation
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.map
+import androidx.lifecycle.viewModelScope
+import com.example.kursaa.core.base.BaseViewModel
+import com.example.kursaa.features.characters.domain.GetCharactersUseCase
+import com.example.kursaa.features.characters.domain.model.Character
+import com.example.kursaa.features.characters.presentation.model.CharacterDisplayable
+
+class CharacterViewModel(private val getCharactersUseCase: GetCharactersUseCase) : BaseViewModel() {
+
+    private val _characters by lazy {
+        MutableLiveData<List<Character>>()
+            .also { getCharacters(it) }
+    }
+
+    val characters: LiveData<List<CharacterDisplayable>> = _characters.map { characters ->
```

Comment:
Jeśli prywatne property jest inicjowane `by lazy`, to dla poprawnego działania to publiczne też powinno być inicjowane w ten sam sposób. W przeciwnym wypadku może to powodować np. problemy z testami, tak jak w tym przypadku zgłoszonym na grupie - https://kurs-architektury.slack.com/archives/C016S129S0H/p1614114757000200.